src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -48,6 +48,28 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
         onBlur: () => onBlur && onBlur(JSON.stringify(localValue.current)),
     };
 
+    if (!designTokens.heading1) {
+        designTokens.heading1 = defaultDesignTokens.heading1;
+    }
+    if (!designTokens.heading2) {
```

Comment:
else if?

Reply:
There could be more than one style missing. With the else if we would lose it.